app/Actions/InitializeGitHubRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Actions;
+
+use App\ConsoleWriter;
+use App\Shell;
+
+class InitializeGitHubRepository
+{
+    use AbortsCommands;
+
+    public const WARNING_FAILED_TO_CREATE_REPOSITORY = 'Failed to create new GitHub repository';
+
+    protected $shell;
+    protected $consoleWriter;
+
+    public function __construct(Shell $shell, ConsoleWriter $consoleWriter)
+    {
+        $this->shell = $shell;
+        $this->consoleWriter = $consoleWriter;
+    }
+
+    public function __invoke()
+    {
+        $ghCommandOptions = config('lambo.store.github');
+        if (! $ghCommandOptions) {
+            return;
+        }
+
+        $this->consoleWriter->logStep('Initializing GitHub repository');
+
+        $process = $this->shell->execInProject("gh repo create --confirm {$this->getRepositoryName()} {$ghCommandOptions}");
+        if (! $process->isSuccessful()) {
+            $this->consoleWriter->warn(self::WARNING_FAILED_TO_CREATE_REPOSITORY);
+            $this->consoleWriter->warnCommandFailed($process->getCommandLine());
+            $this->consoleWriter->showOutputErrors($process->getErrorOutput());
+            return;
+        }
+
+        config(['lambo.store.push_to_github' => true]);
+        $this->consoleWriter->success('Successfully created new GitHub repository');
+    }
+
+    protected function getRepositoryName()
+    {
+        $name = config('lambo.store.project_name');
+        $organization = config('lambo.store.github-org');
+        return $organization ? "{$organization}/{$name}" : $name;
```

Comment:
```suggestion
        $name = config('lambo.store.project_name');
        $organization = config('lambo.store.github-org');
        
        return $organization ? "{$organization}/{$name}" : $name;
```

Reply:
Note to self/Jon: Do we prompt them to add `github-org` to their config in the natural usage flow?

Reply:
We can but there are caveats which is why I left that out for now. There is also a quality-of-life bug in the config file upgrade process that I want to fix before adding new options to the config file.

Imagine I had my current employer (acme) configured in my config. You have contacted me to work on something for Tighten and I create a new project with Lambo. It would be pushed to the acme org not tighten unless I'm paying close attention. That's not something we want. 

Having said that, maybe I allow for `ORGS=tighten,acme,google` in `~/.lambo/config` and upon new project creation, Lambo asks which repo to use?

<pre>
lambo new my-app-for-tighten --github='--private'

You belong to multiple organisations and currently, <strong>my-app</strong> will be created as,
<strong>jonsugar/my-app</strong>. Is this what you want? Please choose your preferred 
organisation for this project. [default: jonsugar/my-app]
[1] Yes, use jonsugar/my-app
[2] Use, tighten/my-app
[3] Use, acme/my-app
[4] Use, google/my-app
2

Ok, using tighten/my-app
</pre>